Gopkg.lock

```diff
@@ -2,7 +2,6 @@
 
 
 [[projects]]
-  digest = "1:b03d71ca2664bf579cf9a72e49ed3d21d16313e686bb12829431f63a8e68330b"
```

Comment:
why do you need to update Gopkg.lock? Jenkins is failing because it thinks it's out of sync.